spec/01-unit/01-db/04-dao_spec.lua

```diff
@@ -160,6 +176,26 @@ describe("DAO", function()
       assert.same(42, row.a)
       assert.same(nil, row.ttl)
     end)
+
+    it("hides fields if strategy returns a column with a value set and it is configured to be hidden in the schema", function()
+      local schema = assert(Schema.new(hidden_attr_schema_definition))
+
+      -- mock strategy
+      local strategy = {
+        select = function()
+          return { a = 42, b = null, u = "data", r = { f1 = 10, f2 = null } }
+        end,
+      }
+
+      local dao = DAO.new(mock_db, schema, strategy, errors)
+
+      local row = dao:select({ a = 42 }, { nulls = true })
+      assert.same(42, row.a)
+      assert.same(null, row.b)
+      assert.same(nil, row.u)
```

Comment:
If the DAO itself does not return hidden fields, how does one actually *use* those fields?

I expected this feature to act on the Admin Api level, not on this one.